wiki/_Sidebar.md

```diff
@@ -5,7 +5,7 @@
 * [Floating Windows](./Floating-Windows.md)
 * [Tabs](./Tabs.md)
 * [Screencasting](./Screencasting.md)
-* [Layer‐Shell Components](./Layer%E2%80%90Shell-Components.md)
+* [Layer‐Shell Components](./Layer‐Shell-Components.md)
```

Comment:
Does this work fine? Pretty sure I copied that URL from GitHub itself or something

Reply:
yeah, it works fine! tbh, i think lychee might not care about this one, https://github.com/becheran/mlc did though.

Reply:
browsers do prefer turning special characters into %55%52%4C%20%65%6E%63%6F%64%69%6E%67 (tl note: url encoding), but special characters work in links anyway.

Reply:
i gave it a try with lychee locally, and the special character can remain special or url encoded. i figure you prefer it to be "safe", so i reverted it to the url encoded version